src/App.tsx

```diff
@@ -17,18 +53,35 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                setQuery={setQuery}
+                applyQuery={applyQuery}
+                isTodoCompleted={isTodoCompleted}
+                setIsTodoComlpeted={setIsTodoComlpeted}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading && <Loader />}
+
+              <TodoList
+                todos={visibleTodos}
+                setSelectedTodo={setSelectedTodo}
+                selectedTodo={selectedTodo}
+              />
```

Comment:
I see an empty table.
Let's render todo list only when `isLoading` false 

![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/84296041/2d10bb96-f2aa-4d57-9684-5eaa9e2f9954)

Reply:
👌fixed